tracks/ruby/exercises/resistor-colors/mentoring.md

```diff
@@ -0,0 +1,33 @@
+Resistor Colors is a side exercise, unlocked by Hello World
+
+### New Concepts
+Array, Array#index, `map`, Constant, chaining methods
+
+### Minimal solution for approval
+```ruby
+
```

Comment:
I suggest removing the whitespace above `class`, but adding an empty line below the header, following default [markdownlint](https://github.com/updownpress/markdown-lint/blob/master/rules/022-blanks-around-headers.md).